### DISCUSSION TITLE
Problemas para leer archivo - Deploy en Heroku

### DISCUSSION THREAD

**Discussion by mbeorlegui** [dds-jv/foro#390] [Consultas de herramientas]
Buenas, estamos teniendo problemas para leer el archivo en el deploy de Heroku.

Nosotros para verificar las contraseñas comunes leemos un archivo `.txt`, el cual se encuentra en `/src/main/resources/public/data/commonly_used_passwords.txt`. El mimo localmente abre bien, pero parece que cuando se quiere abrir con Heroku pincha, porque arroja lo siguiente:

```
Error al abrir archivo, verifique que el mismo se encuentre en /app/src/main/resources/public/data/commonly_used_passwords.txt java.io.FileNotFoundException: /app/src/main/resources/public/data/commonly_used_passwords.txt (No such file or directory)  ...
```

Según lo que entiendo, el deploy lo único que hace es copiar el .jar y ejecutarlo... 
¿hay alguna manera también de incluir este archivo en el deploy para que sea leido por el .jar?
Porque sinceramente no encontré nada en los foros de internet.

**EDIT:** Al parecer el problema no es Heroku, si no que al crear el `.jar`, el mismo no lee correctamente el path del archivo, que está incluido en el propio `.jar`.

**Comment by Gdepablo** [comment]
tenes actualizado el repo donde estas deployeando heroku? igualmente me parece que para esta entrega no es necesario heroku

**Reply by mbeorlegui** [reply]
Buenas!
Estoy deployando siguiendo [este tutorial](https://docs.google.com/document/d/1ezGI4s_Jy13JI7Mn-eMXUOUkq7SmVkIYN74hW__6hXI/mobilebasic).
Tengo actualizado el repositorio de heroku, pues se pushea cada vez que se ejecuta el `mvn deploy...`.
El deploy es en la siguiente entrega ya, según tengo entendido, así que estoy adelantando esto porque no es tan fácil como parece.
